toggle.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file was [added](https://github.com/amezin/gnome-shell-extension-ddterm/commit/e0456a8d8968361fa69065d59a9b07d0e1e71c65#diff-702eeb57673f7a7a4aa1547229d25069e01a60bc975daf536742a423bffd14e9), and then [removed](https://github.com/amezin/gnome-shell-extension-ddterm/commit/834f7aece230944175a65fc22693ac692a2c6551#diff-702eeb57673f7a7a4aa1547229d25069e01a60bc975daf536742a423bffd14e9) on the `master` branch. It should not be in your pull request.

Reply:
Removed.